.github/workflows/security_analysis.yml

```diff
@@ -35,5 +35,5 @@ jobs:
       - name: Vulnerability Scan Go Code
         uses: Templum/govulncheck-action@v1.0.0
         with:
-           go-version: 1.20.6
+           go-version-file: go.mod
```

Comment:
looks like the vuln check doesn't support this, and it defaulted to go 1.20.6, looks like its just using a docker tag for the version so probably just use `1.21` here

Reply:
Darn it. I thought it would change once we do build. I'll change it back :sweat_smile: